rsocket-core/src/main/java/io/rsocket/buffer/Tuple3ByteBuf.java

```diff
@@ -0,0 +1,597 @@
+package io.rsocket.buffer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.Unpooled;
+import io.netty.util.ReferenceCountUtil;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.channels.GatheringByteChannel;
+import java.nio.charset.Charset;
+import org.agrona.BufferUtil;
+
+class Tuple3ByteBuf extends AbstractTupleByteBuf {
+
+  private static final int MEMORY_CACHE_ALIGNMENT = 64;
```

Comment:
same as above - make this a property and default it to 8

Reply:
What does that magic number do?

Reply:
https://stackoverflow.com/questions/381244/purpose-of-memory-alignment

Reply:
done